### PR TITLE
Add `primary_email` getter to `DirectoryUser`

### PR DESCRIPTION
**Pull request by maxdeviant**
This PR adds a `primary_email` getter to the `DirectoryUser` struct for easily accessing the primary email.

### CODE REVIEW

File: src/directory_sync/types/directory_user.rs [hunk 1]
```diff
@@ -241,4 +250,69 @@ mod test {
...
+    #[test]
+    fn it_returns_none_for_the_primary_email_when_the_user_does_not_have_a_primary_email() {
+        let directory_user = DirectoryUser {
+            id: DirectoryUserId::from("directory_user_01E1JG7J09H96KYP8HM9B0G5SJ"),
+            idp_id: "2836".to_string(),
+            directory_id: DirectoryId::from("directory_01ECAZ4NV9QMV47GW873HDCX74"),
+            organization_id: Some(OrganizationId::from("org_01EZTR6WYX1A0DSE2CYMGXQ24Y")),
+            username: Some("marcelina@foo-corp.com".to_string()),
+            emails: vec![DirectoryUserEmail {
+                primary: Some(false),
```
Comment (mthadley) [inline src/directory_sync/types/directory_user.rs:299]: Is it worth adding a test to cover the case when the `primary` field is `None`?

**Review by mthadley — APPROVED** [review]

File: src/directory_sync/types/directory_user.rs (hunk 1 above)
Comment (maxdeviant) [reply src/directory_sync/types/directory_user.rs:299]: It might be beneficial, but most likely only in the short-term.

I'd like to see this data structure get changed on the backend to ensure that the `primary` field always comes back. which would allow us to make this field a `bool` instead of an `Option<bool>`.

**Review by sheldonvaughn — APPROVED** [review]

**Comment by maxdeviant** [comment]
Alright, clearly Bors doesn't like my fork.

Closing in favor of #87.
